ci/ovs_test_setup/testpmd_image/testpmd.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+cd /dpdk || exit 1
+#set container id  as env
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/x86_64-linux-gnu/
+while ! [ -s /etc/podinfo/annotations ]; do
+    echo "annotations not found yet"
+    sleep 1 # throttle the check
+done
+cat /etc/podinfo/annotations
+sleep 10
```

Comment:
why sleep here? what are you waiting for ? since the annotations show up, then the while loop ends and continues

Reply:
annotations key gets populated but values take a while.